src/test/test-string-util.c

```diff
@@ -1168,4 +1168,54 @@ TEST(streq_skip_trailing_chars) {
         assert_se(!streq_skip_trailing_chars("", "f", NULL));
 }
 
+#define TEST_MAKE_CSTRING_ONE(x, ret, mode, expect)                     \
+        do {                                                            \
```

Comment:
braced group?

Reply:
dunno, this is local test code that is not supposed to resolve to anything `({ … })` otoh resolves to something. which we don't really want here. 

but i mean, i doubt this really matters, after all we could just discard what it resolves to.

Reply:
oh it's not for any technical reason, I just like the way they look much more than a do; while